exercises/proverb/example.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function(){
+  const last = arguments[arguments.length - 1];
```

Comment:
I saw the ES6 style in this file, but as it's *internal*, I said nothing. Anyway, I agree with @matthewmorgan, it'll be better to use *plain* ES5 as much as possible. Thanks!